concentration/Scene/BilliardScene.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  BilliardScene.swift
+//  concentration
+//
+//  Created by Roman Yarmoliuk on 30.01.2023.
+//
+
+import SpriteKit
+
+class BilliardScene: SKScene {
+    
+    let matrixSize = 5
+    let matrixMargin: CGFloat = 32
+    let spacing: CGFloat = 4
+    
+    var startPoint = CGPoint()
+    var whiteBall = SKShapeNode()
+    
+    override func didMove(to view: SKView) {
+        
+        self.removeAllChildren()
```

Comment:
I guess no need to remove all children after fixing bug with scene creation in menu